fsinfo/src/fsinfo.c

```diff
@@ -340,7 +331,7 @@ static J4statusPluginContext *
 _j4status_fsinfo_init(J4statusCoreInterface *core)
 {
     const gchar FILESYSTEM[] = "Filesystem";
-    const gchar FORMAT_DEFAULT[] = "${avail} (${p_avail}%)";
+    const gchar FORMAT_DEFAULT[] = "${avail(f.1)}b (${p_avail(f04.1)}%)";
```

Comment:
So if we now have automagical prefixes, should not they be on by default?  If I understood the manual correctly, the correct format is "${avail(b.1)}".

Reply:
Oh good catch.